app.js

```diff
@@ -11,8 +11,29 @@ module.exports = app; // for testing
 var config = {
   appRoot: __dirname // required config
 };
-
+config.swaggerSecurityHandlers = {
+  bearerAuth: function securityHandler1(req, authOrSecDef, scopesOrApiKey, callback) {
```

Comment:
Can you exclude this code for this commit, since it will prevent any requests from working?